docs/user-guide/how_to_use_nodegroup_plugin.md

```diff
@@ -84,6 +84,28 @@ label `nodeGroupAffinity.requiredDuringSchedulingIgnoredDuringExecution` or `nod
 kubectl get po job-1-nginx-0 -o wide 
 ```
 
+### Nodegroup Plugin Strict Configuration
+
+Strict is a switch to specific behavior of unset affinity queue. If `strict: false`, workload assigned to a unset affinity queue could be scheduled to Node with empty label `volcano.sh/nodegroup-name`. Otherwise, workload cannot be scheduled.
```

Comment:
Seems that it's `if the node doesn't have the `volcano.sh/nodegroup-name` label or the queue doesn't set affinity` ?